step5/copy5-1.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+//6.1インターフェース　関数にインターフェースを埋め込む
+//https://docs.google.com/presentation/d/1vKdJHHx4A_sP_Ft-3cakQqfJRbKxWSI9axzQ-GHHOoY/edit#slide=id.g4f15a7d687_0_214
+
+/*
+インターフェースの実用に近い例は、step3/pointer_reciever_sample.goを参照
```

Comment:
ここGithubのURLにしておくと良いかも！

```suggestion
インターフェースの実用に近い例は、 https://github.com/CATechAccel/learn-golang/blob/feat/8/FunctionAndType/step3/pointer_reciever_sample.go を参照
```